doc/data/messages/u/unsupported-delete-operation/good.py

```diff
@@ -1 +1,3 @@
-# This is a placeholder for correct code for this message.
+FRUITS = ("apple", "orange", "berry")
+
+del list(FRUITS)[0]
```

Comment:
`list(FRUITS)` is immediately discarded, so I would consider constructing an example with a more realistic use of `del`.